src/Cron/Legacy/cron.ctrlPlanning.php

```diff
@@ -19,28 +19,25 @@
 
 @note : Modifiez le crontab de l'utilisateur Apache (ex: #crontab -eu www-data) en ajoutant les 2 lignes suivantes :
 # Controle du planning du lundi au vendredi à 7h
-0 7 * * 1-5 /usr/bin/php5 -f /var/www/html/planning/cron.ctrlPlannings.php
+0 7 * * 1-5 /usr/bin/php -f /var/www/html/planning/src/Cron/Legacy/cron.ctrlPlannings.php
 Remplacer si besoin le chemin d'accès au programme php et le chemin d'accès à ce fichier
-@note : Modifiez la variable $path suivante en renseignant le chemin absolu vers votre dossier planningBiblio
 */
 
-$path="/var/www/html/planning";
-
 /** $version=$argv[0]; permet d'interdire l'execution de ce script via un navigateur
  *  Le fichier config.php affichera une page "accès interdit si la $version n'existe pas
  *  $version prend la valeur de $argv[0] qui ne peut être fournie que en CLI ($argv[0] = chemin du script appelé en CLI)
  */
 
 $version=$argv[0];
 
-// chdir($path) : important pour l'execution via le cron
-chdir($path);
+require_once(__DIR__ . '/../../../vendor/autoload.php');
+
+use App\PlanningBiblio\Framework;
 
-require_once "$path/include/config.php";
-require_once "$path/include/function.php";
-require_once "$path/absences/class.absences.php";
-require_once "$path/planning/postes_cfg/class.tableaux.php";
-require_once "$path/postes/class.postes.php";
+require_once(__DIR__ . '/../../../public/include/config.php');
+require_once(__DIR__ . '/../../../public/include/function.php');
+require_once(__DIR__ . '/../../../public/absences/class.absences.php');
+require_once(__DIR__ . '/../../../public/postes/class.postes.php');
 
 
 // Génération d'un CSRF Token
```

Comment:
TEST : 
<pre>PHP Warning:  session_start(): Cannot start session when headers already sent in /var/www/html/plb/src/Cron/Legacy/cron.ctrlPlanning.php on line 44</pre>

En supprimant session_start(); en ligne 44 : ça passe, et le reçoit bien les mails.